lib/bolt/pal/yaml_plan/step/resources.rb

```diff
@@ -65,6 +65,7 @@ def body
 
           def transpile
             code = StringIO.new
+            code.print "  apply_prep(#{Bolt::Util.to_code(@target)})\n"
```

Comment:
This should use the `Step#function_call` helper.

Reply:
Updated. That never even crossed my mind. Thanks.